docs/src/tutorials/sample/material_solvated.ipynb

```diff
@@ -77,17 +80,22 @@
    "source": [
     "For information about the data being read in and the details of the model see [the previous tutorial](../fitting/simple_fitting.rst). \n",
     "We will gloss over these details here.\n",
-    "Use link to [download](example.ort) the ort data."
+    "We use `pooch` to fetch the file from the repository."
    ]
   },
   {
    "cell_type": "code",
-   "execution_count": null,
+   "execution_count": 3,
```

Comment:
This line and line 28 show that the notebook cells have been run before committing. Ideally, we want the notebook to have all outputs cleared.

Reply:
Thanks, should be clean now 🧼